src/general/GameWorld.cs

```diff
@@ -338,8 +345,6 @@ public void ReplacePlayerSpeciesWith(Species species, bool deleteOldPlayer = tru
     /// </summary>
     public void GenerateRandomSpeciesForFreeBuild(long seed = 0)
     {
-        Random random;
```

Comment:
I think this is a bad change, the freebuild stuff here recreates the random so it is kind of weird how it is a field. I'd not make the random a field unless you consider these overall implications more for this class.